sklearn/cross_validation.py

```diff
@@ -332,6 +332,119 @@ def __len__(self):
         return self.n_folds
 
 
+def disjoint_label_folds(y, n_folds=3):
```

Comment:
We could call it labels, y is simply for consistency with other cross-validation methods (.e.g. StratifiedKFold).